linux_os/guide/system/software/integrity/crypto/configure_bind_crypto_policy/rule.yml

```diff
@@ -9,6 +9,8 @@ description: |-
     BIND is supported by crypto policy, but the BIND configuration may be
     set up to ignore it.
 
+    Either BIND is not installed, or it is configured to use the system-wide defined crypto policy.
```

Comment:
This doesn't go here. This goes into OCIL.

Reply:
Moved to OCIL.

Reply:
Relocated to OCIL clause.